stable/vault-operator/values.yaml

```diff
@@ -4,7 +4,7 @@ replicaCount: 1
 image:
   repository: quay.io/coreos/vault-operator
   tag: 0.1.9
-  pullPolicy: IfNotPresent
```

Comment:
I think `IfNotPresent` makes more sense for a stable version, rather update the docs to match.

Reply:
I second that. The default should stay as is: `IfNotPresent`. The readme should be fixed instead.